packages/dd-trace/src/ci-visibility/exporters/ci-visibility-exporter.js

```diff
@@ -37,6 +37,9 @@ class CiVisibilityExporter extends AgentInfoExporter {
       }
     })
 
+    // This should be overwritten by the class extending CiVisibilityExporter
+    this._apiUrl = this._url
```

Comment:
It looks like something that we will forget to change...
Can't we use a getter and override it in the extension if it is necessary?

```
_getApiUrl () {
  return this._url
}
```

Reply:
that's a good idea

Reply:
done in https://github.com/DataDog/dd-trace-js/pull/2664/commits/aeb8f0701ec5606c3cc1b3ac697054b90d544408. Used the chance to do something similar to `setUrl`